index.html

```diff
@@ -2713,7 +2713,913 @@ <h5>Question</h5>
+                  </p>
+              </section>
+              <section>
+                  <h5>Responses</h5>
+                  <ol type="A">
+                      <li>
+                          Each DID is entirely isolated; there is no practical way to infer relationships of common control between them.
+                      </li>
+                      <li>
+                          It may be possible to infer relationships of common control based on timing and the IP address of clients, but not based on data persisted in the VDR itself. Inference is expensive or impractical.
+                      </li>
+                      <li>
+                          It is possible to infer relationships based on data persisted in the VDR itself, and the inference is easy.
+                      </li>
+                  </ol>
+              </section>
+              <section>
+                  <h5>Relevance</h5>
+                  <p>
+                      Inferring relationships between Bitcoin addresses allowed law enforcement to track down the operators of Silk Road, even though those operators believed they had privacy.
+                  </p>
+              </section>
+              <section>
+                  <h5>Examples</h5>
+                  <table class="simple">
+                      <tr>
+                          <th>
+                              Method
+                          </th>
+                          <th>
+                              Net.
+                          </th>
+                          <th>
+                              Reg.
+                          </th>
+                          <th>
+                              Notes
+                          </th>
+                      </tr>
+                      <tr>
+                          <td>
+                              did:foo
+                          </td>
+                          <td>
+                              A
+                          </td>
+                          <td>
+                              D
+                          </td>
+                          <td>
+                              By design, did:foo is super private.
+                          </td>
+                      </tr>
+                  </table>
+              </section>
+          </section>
+          <section>
+              <h4>Incentives for Multicontext DIDs</h4>
+              <section>
+                  <h5>Question</h5>
+                  <p>
+                      To what extent does the method incentivize (due to cost, hassle, etc) a DID to be reused in multiple contexts?
+                  </p>
+              </section>
+              <section>
+                  <h5>Responses</h5>
+                  <ol type="A">
+                      <li>
+                          The natural and default behavior is to create a new DID in each new context, and there is no meaningful incentive to do otherwise.
```

Comment:
```suggestion
The natural and default behavior is to create a new DID in each new
context, and there is no meaningful incentive to do otherwise.
```